modules/oceanbase-ce/src/test/java/org/testcontainers/junit/oceanbase/SimpleOceanBaseTest.java

```diff
@@ -0,0 +1,66 @@
+package org.testcontainers.junit.oceanbase;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.OceanBaseTestImages;
+import org.testcontainers.containers.OceanBaseContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.db.AbstractContainerDatabaseTest;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SimpleOceanBaseTest extends AbstractContainerDatabaseTest {
```

Comment:
Can a test using grpc be added?

Reply:
Sorry but the database does't support grpc, it uses a custom rpc protocol to communicate with other database servers or tools like [obproxy](https://github.com/oceanbase/obproxy) or [oblogproxy](https://github.com/oceanbase/oblogproxy).